lib/_http_client.js

```diff
@@ -374,9 +375,12 @@ function socketCloseListener() {
     // receive a response. The error needs to
     // fire on the request.
     req.socket._hadError = true;
+    req.emit('aborted');
     req.emit('error', createHangUpError());
```

Comment:
I'm not sure `error` makes sense here. But changing that would be a sem-major change.

Reply:
I think aborted should at least be emitted after error? Anyone else?

Reply:
I'm 👎  in not emitting an error here.

Reply:
@mcollina: aborted after or before error?

Reply:
aborted after.

Reply:
Fixed